src/components/NavBar.js

```diff
@@ -0,0 +1,17 @@
+import {NavLink} from "react-router-dom"
+import "./NavBar.css"
+
+const NavBar = () => {
+    return (
+        <header>
+            <nav>
+                <NavLink to={"/"}><img src={"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQh3H2SCuJBcmnrjvLMYrbWth4-_pUKiPUwdQ&usqp=CAU"}></img></NavLink>
```

Comment:
Add an exact to the / NavLink so the active class isn't triggered on all routes.